src/include/network/lan/udev.rb

```diff
@@ -45,6 +45,30 @@ def update_udev_rule_key(rule, key, value)
       rule
     end
 
+    # Writes new persistent udev net rules and tells udevd to update its configuration
+    def write_update_udevd(udev_rules)
+      SCR.Write(path(".udev_persistent.rules"), udev_rules)
+      SCR.Write(path(".udev_persistent.nil"), [])
+
+      update_udevd
+    end
+
+    # Tells udevd to reload and update its configuration
+    #
+    # @return [boolean] false when new configuration cannot be activated
+    def update_udevd
+      SCR.Execute(path(".target.bash"), "udevadm control --reload")
+
+      # When configuring a new s390 card, we neglect to fill
+      # its Items[i, "udev", "net"], causing jumbled names (bnc#721520)
+      # The udev trigger will make udev write the persistent names
+      # (which it already has done, but we have overwritten them now).
+      SCR.Execute(
+        path(".target.bash"),
+        "udevadm trigger --subsystem-match=net --action=add"
+      ) == 0
```

Comment:
this is not much obvious, can you please use `retcode = SCR.Execute...` and then `retcode == 0` this way it can be easy to overlook